tests/foreman/longrun/test_inc_updates.py

```diff
@@ -147,8 +150,13 @@ def host(
     rhel7_contenthost_module.execute(f'yum install -y {FAKE_4_CUSTOM_PACKAGE}')
     rhel7_contenthost_module.execute('katello-package-upload')
     host = entities.Host().search(query={'search': f'name={rhel7_contenthost_module.hostname}'})
-    # Force host to generate or refresh errata applicability
-    call_entity_method_with_timeout(host[0].errata_applicability, timeout=600)
+    # Disabling call to timout due to BZ#2017533, but if we get no failures, we can remove it.
+    if not is_open('BZ:2017533'):
+        # Force host to generate or refresh errata applicability
+        call_entity_method_with_timeout(host[0].errata_applicability, timeout=120)
+    else:
+        host[0].errata_applicability(synchronous=False)
```

Comment:
Won't the async applicability effect in the later stage of this test?

Reply:
can you elaborate on this?

Reply:
@jyejare please? :D

Reply:
We are trying to async the `errata applicability` step which might run in the BG while we move to the next steps and if it didn't complete before the next step/steps where the next step might need errata applicability to complete then we will end up with failing step.

This could affect next step / next test as well if its not sync, I mean we should wait to finish the errata applicability in any case !

Reply:
that's a very good point. we could also set synchronous to True.

Reply:
Hi, so it seems to me we need to implement a "wait for tasks" type procedure within Nailgun itself. i.e. use API to search for repo publish task, using the name of repo being published not ID of task which we no longer have.

Reply:
Hi,  it seems to me we need to implement a "wait for tasks" type procedure within Nailgun itself. i.e. use API to search for repo publish task, using the name of repo being published not ID of task which we no longer have. 

But please note there is no more `Actions::Katello::Host::GenerateApplicability` task to search for, so for this `errata applicability` case I don't see a way to check for completion of that errata enumeration task, and we don't want to use sleep. I will ask Devs for more ideas, but  I believe the errata applicability is now faster as its not a foreman task.